doc/rbd/rbd-live-migration.rst

```diff
@@ -301,6 +301,19 @@ as follows::
   stored in the config-key store via ``ceph config-key set <key-path> <value>``
   (e.g. ``ceph config-key set rbd/s3/access_key NX5QOQKC6BH2IDN8HC7A``).
 
+The ``nbd`` stream can be used to import from a remote NBD server. Its
+``source-spec`` JSON is encoded as follows::
+
+        {
+            <format unique parameters>
+            "stream": {
+                "type": "nbd",
+                "url": "<ip>",
```

Comment:
I called it 'server' since you may provide IP or the server name.

Reply:
I am not sure I like 'server', just because I don't recall I saw using such a name, and I would want something commonly used. But I don't have a strong opinion, you may leave it as you wish, probably other reviewers will have a suggestion.